tools/test-setup.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+# This tool is used to setup the environment for running the tests. Its name
+# name and location is based on Zuul CI, which can automatically run it.
+set -euo pipefail
+
+# User specific environment
+# shellcheck disable=SC2076
+if ! [[ "$PATH" =~ "$HOME/.local/bin" ]]; then
+    PATH="$HOME/.local/bin:$PATH"
+fi
+
+if [ -f "/usr/bin/apt-get" ]; then
+    if [ ! -f "/var/cache/apt/pkgcache.bin" ]; then
+        sudo apt-get update # mandatory or other apt-get commands fail
+    fi
+    # avoid outdated ansible and pipx
+    sudo apt-get remove -y ansible pipx || true
```

Comment:
Where is `ansible` comming from? is this something that came from the forking from upstream? It dont seems to be relevant in our use-case in this scenario because we do not use ansible or have it installed at all?